assets/components/openshift-router/serving-certificate.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-ingress
+  name: router-certs-default
```

Comment:
Should this name match the change in service-internal.yaml?

Reply:
it shouldn't, these are two different secrets

Reply:
This is more aligned with what OCP has, right?

An OCP 4.12 cluster has the following objects:

```
 oc --kubeconfig kubeconfig-clusterbot get secret router-certs-default -n openshift-ingress
NAME                   TYPE                DATA   AGE
router-certs-default   kubernetes.io/tls   2      41m
```

```
[root@maxwell ~]# oc --kubeconfig kubeconfig-clusterbot get svc router-internal-default -n openshift-ingress -o yaml
apiVersion: v1
kind: Service
metadata:
  annotations:
    service.alpha.openshift.io/serving-cert-secret-name: router-metrics-certs-default
    service.alpha.openshift.io/serving-cert-signed-by: openshift-service-serving-signer@1667219307
    service.beta.openshift.io/serving-cert-signed-by: openshift-service-serving-signer@1667219307
  creationTimestamp: "2022-10-31T12:29:41Z"
  labels:
    ingresscontroller.operator.openshift.io/owning-ingresscontroller: default
  name: router-internal-default
  namespace: openshift-ingress
  ownerReferences:
  - apiVersion: apps/v1
    controller: true
    kind: Deployment
    name: router-default
    uid: 158f6632-1e72-42e1-a455-e3412ac5666a
  resourceVersion: "9123"
  uid: daded2dc-55fd-41b1-b690-a46e11a5b171
spec:
  clusterIP: 172.30.158.236
  clusterIPs:
  - 172.30.158.236
  internalTrafficPolicy: Cluster
  ipFamilies:
  - IPv4
  ipFamilyPolicy: SingleStack
  ports:
  - name: http
    port: 80
    protocol: TCP
    targetPort: http
  - name: https
    port: 443
    protocol: TCP
    targetPort: https
  - name: metrics
    port: 1936
    protocol: TCP
    targetPort: 1936
  selector:
    ingresscontroller.operator.openshift.io/deployment-ingresscontroller: default
  sessionAffinity: None
  type: ClusterIP
status:
  loadBalancer: {}
```